File: ABC155/ABC155A.hs
import qualified Data.ByteString.Char8 as BS
import qualified Data.Maybe as MA
import qualified Data.List as L

main = do
    xs <- L.sort . fmap (fst . MA.fromJust . BS.readInt) . BS.words <$> BS.getLine
    putStrLn $ if L.length (L.nub xs) == 2 then "Yes" else "No"

File: ABC155/ABC155A.c
#include <stdio.h>

int main(){
    int a, b, c;

    scanf("%d %d %d\n", &a, &b, &c);

    if((a != b && b != c && c != a) || (a == b && b == c )){
        printf("No\n");
    }else{
        printf("Yes\n");
    }
}
